pandas/tests/indexing/test_at.py

```diff
@@ -47,6 +37,67 @@ def test_selection_methods_of_assigned_col():
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize(
```

Comment:
General comment - #30649 issue talks exclusively about assigning a list of values using `.at`. I went through this file though and haven't found a general test that makes sure that other values (e.g. tuples, strings, nans etc.) work as expected, so I thought that I will create such a general test for both df and series (including the list, of course)

Reply:
expansion of tests like this is great!